backend/shared/src/resolve-market-helpers.ts

```diff
@@ -96,6 +96,9 @@ export const resolveMarketHelper = async (
     const hasAnswerResolvedYes =
       unresolvedContract.answers.some((a) => a.resolution === 'YES') ||
       outcome === 'YES'
+    const marketCancelled = unresolvedContract.answers
+      .every((a) => a.resolution === 'CANCEL')
+    const finalResolution = marketCancelled ? 'CANCEL' : 'MKT'
```

Comment:
I would move this inside the if statement. But it doesn't matter that much so I'll merge it